project-plan.md

```diff
@@ -0,0 +1,189 @@
+
+There are two key areas of interoperability:
+1. Interoperability between different ledgers - where DIDs and VCs are stored.
+2. Interoperability between different wallets - where the ownership of DIDs and VCs are prooved.
+
+## 1.3 Rammer
+
+The BSc project runs between `01.feb.2021` and `20.may.2021`.
+
+The BSc project will be developed as an open source project on the DIN-Foundation's Github page here: https://github.com/DIN-Foundation/bsc-ntnu-2020/. 
+
+See https://www.din.foundation/ for more info.
+
+
+# 2. Omfang
+
+## 2.1 Fagområde
+
+The project is developed within the field of Self-sovereign-identity, aka SSI.
+
+## 2.2 Avgrensing
+
+The project is looking at interoperability of the current SSI-infrastructure. More specifically interoperability between wallets, and interoperability between ledgers.
+
+To understand interoperability a proof-of-concept application will be developed. The proof-of-concept will implement one specific use-case of SSI. 
+
+The proof-of-concept should implement the following parts of an SSI-workflow:
+- Issuing a credential to a ledger
+- Hold the credential in a wallet
+- Proove ownership of a credential to a 3rd party.
+
+A limited set of ledgers and wallets should be investigated:
+
+**@TODO Make list of wallets to investigate**
+
+**@TODO Make list of ledgers to investigate**
+
+## 2.3 Oppgavebeskrivelse
+
+*See [project-description.md](./project-description.md)*
+
+### User story
+
+*Disclaimer: The BSc has no affiliation with Statens Vegvesen at the moment of writing, and the story below is pure fantasy.*
+
+The norwegian driver license issuer, Statens Vegvesen, is considering to start issuing it's driver licenses as verifiable credentials (VCs). Statens Vegvesen is not sure if verifiable credentials is the future yet, but are willing to try and dip it's toes in the water. Statens vegvesen will still issue credentials in the traditional way for the forseeable future.
+
+What Statens Veivesen want is a proof-of-concept SSI-application which will issue, hold and verify driver-licenses. They are hoping that this will bootstrap SSI in Norway, as this will enable other individuals and organizations to start experimenting with holding and verifying a serious credential which actually is useful.
+
+The proof-of-concept may demonstrate that a driver license as a verifiable credential could be considered on par, legally speaking, with traditional driver licenses.
+
+Statens Vegvesen wants the application to follow open standards which will enable the application to be agnostic about where and how credentials are issued, stored and verified. In other words Statens Vegvesen want to avoid the application to be locked to a specific ledger and a specific wallet.
+
+
+### Deliverables
+
+- Proof-of-concept SSI application which implements the user-story
+- A comparison matrix of ledger interoperability
```

Comment:
You might want to have a small research round to see what is worth doing matrices on, or leaving this field a bit open while you journey through the space.

Suddenly you can have something that is worth a comparison matrix